model.go

```diff
@@ -1365,6 +1399,16 @@ func (m *model) validateSecrets(validationCtx *validationContext) error {
 	return nil
 }
 
+// validateVirtualHostKeys validates any virtual host keys
```

Comment:
This is missing period and also doesn't explain what it's validating, that it exists? That it is an expected algorithm?

Reply:
This probably doesn't need the docstring. The exported Validate() method one would be sufficient.

Reply:
Remove the docstring